news/128.new.rst

```diff
@@ -0,0 +1,5 @@
+Add a hook for ``spacy`` which contains hidden imports and data files
+
+Add hooks for ``thinc`` and ``thinc.banckends.numpy_ops`` which contain data files and hidden imports. These hooks were created to make spacy work correctly.
+
+Added hook for ``srsly.msgpack._packer`` which contains a hidden import.
```

Comment:
There's another thing to add to #130. The correct way to have multiple towncrier news items in one pull request is to put them in separate files but add a `.[sub-enumeration]` to their names. In this case it the filesnames should be:
- `128.0.new.rst`
- `128.1.new.rst`
- `128.2.new.rst`